backend/src/lib/crypto/srp.ts

```diff
@@ -108,17 +114,18 @@ export const getUserPrivateKey = async (password: string, user: TUserEncryptionK
     raw: true
   });
   if (!derivedKey) throw new Error("Failed to derive key from password");
-  const key = decryptSymmetric({
-    ciphertext: user.protectedKey!,
-    iv: user.protectedKeyIV!,
-    tag: user.protectedKeyTag!,
-    key: derivedKey.toString("base64")
+  const key = decryptSymmetric128BitHexKeyUTF8({
```

Comment:
is this key always 128 bit or can it be 256 bit too?

Reply:
@dangtony98 named this. I don't thin we have 256 bit key.

CC: @dangtony98